src/community/infix-short-scale.ts

```diff
@@ -0,0 +1,25 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import { toSubscript, abbreviate } from "../utils";
+import Decimal from "break_infinity.js";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixShortScaleNotation extends AbstractInfixNotation {
+  public get name(): string {
+    return "Infix short scale";
+  }
+  
+  protected canHandleZeroExponent = false;
+
+  protected formatMantissa(digit: number): string {
+    return toSubscript(digit);
+  }
+  protected formatExponent(exp: number): string {
+    if (exp < 0) {
+      return (exp / 3).toString();
+    }
+    return abbreviate(exp / 3 - 1);
+  }
+  public formatNegativeDecimal(value: Decimal, places: number): string {
```

Comment:
Codacy found an issue: [Declaration of public instance method not allowed after declaration of protected instance method. Instead, this should come after protected instance fields.](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)